imap_processing/codice/utils/codice_utils.py

```diff
@@ -1,5 +1,57 @@
+"""Various classes and functions used throughout CoDICE processing.
+
+This module contains utility classes and functions that are used by various
+other CoDICE processing modules.
+"""
+
+import collections
+import dataclasses
 from enum import IntEnum
 
+import xarray as xr
+
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.codice import codice_cdf_attrs
+
+# Placeholder for the list of mnemonics to convert from analog to engineering units
+CONVERT_TO_EU_LIST = []
+
+
+class CODICEAPID(IntEnum):
```

Comment:
Will all of these be needed? Nice job!

Reply:
Probably not -- I remember Joey mentioning that in reality we wont get some of these, but it was easy enough to just put them all here for now.